.github/workflows/user_geneset_tests.yml

```diff
@@ -1,21 +1,55 @@
-
 name: User Geneset Tests
 
 on:
   push: {}
   pull_request:
     branches:
-      - master
+      - test_github_actions
 
 jobs:
-  run_app_tests:
+  run_user_geneset_tests:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ 3.6, 3.7, 3.8, 3.9, 3.10 ]
+        python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11" ]
+    services:
+      elasticsearch:
+        image: docker.elastic.co/elasticsearch/elasticsearch:8.5.2
+        env:
+          discovery.type : single-node
+          xpack.security.enabled: false
+          xpack.security.http.ssl.enabled: false
+          xpack.security.transport.ssl.enabled: false
+        options: >-
+          --health-cmd "curl http://localhost:9200/_cluster/health"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 10
+        ports:
+          - 9200:9200
     steps:
-      - name: Checkout source
+      - name: Create ES Index
+        run: |
+          sleep 60
+          curl -X PUT http://localhost:9200/user_genesets
+      - name: Clone frontend repo
+        uses: actions/checkout@v3
+        with:
+          repository: biothings/mygeneset.info-website
+          path: mygeneset.info-website
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          working-directory: mygeneset.info-website
+          node-version: "16"
+      - name: Install packages
+        run: yarn install
+        working-directory: mygeneset.info-website
+      - name: Build app
+        run: yarn build
+        working-directory: mygeneset.info-website
+      - name: Checkout MyGeneset source
         uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
```

Comment:
@everaldorodrigo I believe both `actions/checkout` and `actions/setup-python` here have new versions to use.